src/components/image.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from "react"
+
+interface Props {
+  image: string
+  className?: string
+  onLoad?: () => void
+  getImageAltAttribute?: string
```

Comment:
In terms of naming, it sounds like a function that is passed but it's just the attribute. Can we simply name it `alt` in this case?